src/xwayland.c

```diff
@@ -497,8 +497,13 @@ xwayland_view_maximize(struct view *view, bool maximized)
 		maximized);
 }
 
+enum direction {
+	LAB_TO_FRONT,
+	LAB_TO_BACK,
+};
```

Comment:
Even though this is an xwayland.c internal enum I'd propose something more specific so we don't end up clashing with another `direction` enum in the future, bad examples: `z_order` or `stack_direction`.

Reply:
Good point. Done.